intl.emu

```diff
@@ -87,7 +87,7 @@
         <h1>
           FormatNumericToString (
```

Comment:
You can delete all the AOs that now have no diff.

Reply:
Done